release_notes.md

```diff
@@ -11,3 +11,4 @@
 - Update PowerShell Worker 7.2 to 4.0.2803 [Release Note](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v4.0.2803)
 - Update PowerShell Worker 7.4 to 4.0.2802 [Release Note](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v4.0.2802)
 - Fixing bug with placeholder misses in dotnet-isolated #9253
+- Update PowerShell Worker 7.0 to 4.0.2823 [Release Note](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v4.0.2823)
```

Comment:
Can you please remove the previous entry for the PowerShell worker? The one you're adding would be the valid one for this release

Reply:
Disregard. I see those are different workers

Reply:
As per our conversation offline, we will keep this as is. Resolving.